src/nifreeze/model/base.py

```diff
@@ -69,7 +69,7 @@ def init(model: str | None = None, **kwargs):
 
             return AverageDWIModel(kwargs.pop("dataset"), **kwargs)
 
-        if model.lower() in ("gqi", "dti", "dki", "pet"):
+        if model.lower() in ("gqi", "dti", "dki"):
             from importlib import import_module
 
             dmrimod = import_module("nifreeze.model.dmri")
```

Comment:
Would it make more sense to keep the "pet" keyword and make this more generic?

```suggestion
            thismod = import_module(f"nifreeze.model.{'pet' if model.lower() == 'pet' else 'dmri'}")
```